apps/site/src/components/icons/mapbox-logo-icon.tsx

```diff
@@ -0,0 +1,58 @@
+import { SvgIcon, SvgIconProps } from "@mui/material";
```

Comment:
I don't really mind, happy to use the `public/assets` from your approach once that PR has merged.

Reply:
It has now been merged, if you want to switch

Reply:
I've thought about it a bit more, and I think I prefer the react component approach because the SVG component is served as part of the DOM itself instead of using an HTML `img` component to render the SVG. I've therefore removed the icons from the `assets` folder in https://github.com/blockprotocol/blockprotocol/pull/900/commits/2e806b21cf3bff09c9d3664fecea0e863ed53116